src/lib/components/primitives/message/input-message.svelte

```diff
@@ -1,11 +1,15 @@
 <script lang="ts">
+  import sanitize from 'xss';
   import type { Maybe } from '$lib/interfaces';
 
-  export let message: string;
+  export let classes = '';
```

Comment:
This was a really simple solution to eliminate the `<span>` that was picked up on earlier - thanks @cerebrl 👍